lichess-bot.py

```diff
@@ -102,7 +101,7 @@ def game_logging_configurer(queue, level):
 def start(li, user_profile, engine_factory, config, logging_level, log_filename, one_game=False):
     challenge_config = config["challenge"]
     max_games = challenge_config.get("concurrency", 1)
-    logger.info("You're now connected to {} and awaiting challenges.".format(config["url"]))
+    logger.info(f'You\'re now connected to {config["url"]} and awaiting challenges.')
```

Comment:
This change is strange. Why not use single quotes for `config['url']` and keep the outer double quotes? It gets rid of an escaping backslash.